### PR TITLE
Add initial notifier support.

### PR DESCRIPTION
**Pull request by mrstegeman**
References mozilla-iot/gateway#1758

### CODE REVIEW

**Comment by mrstegeman** [comment]
@hobinjk I have changes to the gateway and email-sender-adapter as well (for testing), but both will depend on this PR. I can go ahead and put those up if you want to try them all out together.

**Comment by mrstegeman** [comment]
Adapter: https://github.com/mozilla-iot/email-sender-adapter/tree/notifier
Gateway: mozilla-iot/gateway#1857

File: lib/constants.js [hunk 1]
```diff
@@ -69,3 +78,9 @@ exports.THING_TYPE_ON_OFF_LIGHT = 'onOffLight';
 exports.THING_TYPE_DIMMABLE_LIGHT = 'dimmableLight';
 exports.THING_TYPE_ON_OFF_COLOR_LIGHT = 'onOffColorLight';
 exports.THING_TYPE_DIMMABLE_COLOR_LIGHT = 'dimmableColorLight';
+
+exports.NotificationLevel = {
+  LOW: 0,
+  NORMAL: 1,
+  HIGH: 2,
```
Comment (hobinjk) [inline lib/constants.js]: I should know this already, but is there a similar set of urgency levels for alarm events we could mirror here?

Comment (mrstegeman) [reply lib/constants.js]: No, we never decided on levels for the alarm property/event, so it doesn't exist in the schema.

**Pull request merged**